lms/templates/verify_student/face_photo_step.underscore

```diff
@@ -1,8 +1,22 @@
 <div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+
+    <div class="requirements">
+        <h3 class="title"><%- gettext( "What You Need for Verification" ) %></h3>
```

Comment:
now that I've seen the structure of the page, this should be an h2, and the two children should be h3.